lib/web_ui/test/gradient_test.dart

```diff
@@ -17,7 +16,7 @@ void testMain() {
     expect(
       Gradient.radial(
           Offset.zero,
-          null,
+          5.0,
```

Comment:
AFAICT it means we're not passing `focal` and `focalRadius`. The reason I'm passing 5.0 instead of null is because `Gradient.radial` doesn't allow null radius, so I had to pick some value. 5.0 seemed as good as any :)